hardware_interface/src/resource_manager.cpp

```diff
@@ -429,6 +431,11 @@ class ResourceStorage
   void import_command_interfaces(HardwareT & hardware)
   {
     auto interfaces = hardware.export_command_interfaces();
+    hardware_info_map_[hardware.get_name()].command_interfaces = add_command_interfaces(interfaces);
+  }
+
+  std::vector<std::string> add_command_interfaces(std::vector<CommandInterface> & interfaces)
```

Comment:
a little bit of documentation would be nice on this function, how it consumes it output and adds it to certain structures and how the return value makes sense still. it's an internal one, yes but still should be documented for maintainability